llvm/lib/CodeGen/AsmPrinter/AsmPrinter.cpp

```diff
@@ -2764,6 +2770,27 @@ void AsmPrinter::emitJumpTableInfo() {
     for (const MachineBasicBlock *MBB : JTBBs)
       emitJumpTableEntry(MJTI, MBB, JTI);
   }
+
+  if (EmitJumpTableSizesSection && TM.getTargetTriple().isOSBinFormatELF() &&
```

Comment:
Outside of flags and section name, this is pretty generic. Why not open this to all object fully formats?

Reply:
I've added support for COFF (hopefully correctly). I'm completely unfamiliar with MachO so I suggest that we leave that for a followup.